app/src/main/java/org/p2p/wallet/auth/model/PhoneNumber.kt

```diff
@@ -0,0 +1,20 @@
+package org.p2p.wallet.auth.model
+
+data class PhoneNumber(val value: String) {
+
+    object PhoneNumberFormatter {
+
+        fun getPhoneNumberE164Formatted(phoneNumber: PhoneNumber): String {
+            val trimmedValue = phoneNumber.value.replace(" ", "")
+            return "+$trimmedValue"
+        }
+    }
+}
+
+fun PhoneNumber.e164Formatted(): String {
+    return PhoneNumber.PhoneNumberFormatter.getPhoneNumberE164Formatted(this)
+}
+
+fun PhoneNumber.equals(phoneNumber: PhoneNumber?): Boolean {
```

Comment:
!!! please override equals inside PhoneNumber, i don't think == != will work with this method 

e164Formatted can be a method too